docs/src/literate-tutorials/maxwell_good_bad_ugly.jl

```diff
@@ -9,9 +9,9 @@ solution for a Maxwell problem, when vectorized `Lagrange` interpolations conver
 
 ![Results for different discretizations](maxwell.png)
 
-**Figure 1**: The results of this tutorial, showing how the analytical solution is not found when discretizing
+**Figure 1**: The results of this tutorial, showing how the exact solution is not found when discretizing
```

Comment:
Stylistic detail, but maybe we can get rid of the negation here. Maybe something like:
```suggestion
**Figure 1**: The results of this tutorial, showing how the approximation converges to the wrong solution when discretizing
```